packages/svelte-materialify/@types/Select.d.ts

```diff
@@ -0,0 +1,45 @@
+import { SvelteComponent } from './shared';
+
+interface SelectProps<T extends number[] | string[] = string[]> {
+  /** Classes to add to select wrapper. */
+  class?: string;
+  /** Whether select is opened. */
+  active?: string;
```

Comment:
active is a boolean, I believe

Reply:
Thanks for the review. I fixed these issues, and some more. Generic prop typing seemed like a good idea, but it does not work in practice, so I removed it.